### ISSUE TITLE
Training setup for ablation experiments

### ISSUE THREAD

**Issue by harihp12** [facebookresearch/dino#202]
Thanks for the interesting work! Can you explain the training setup for the ablation experiments using ViT-Small with patch size 16 in appendix D of the paper? I am specifically interested in the ablation involving "sharpening" and "longer training" (attaching relevant tables below). In general, I believe these trainings use 100 epochs (is that correct?). Are there any other training settings that are different from the standard DINO ViT-Small-16 training? I expected the kNN-top1 accuracy in the "longer training" ablation table for 100 epoch to be 69.7 (highlighted value in many other ablation tables). But the paper reports 70.9. Maybe something else is different between these trainings. Can you clarify?

**Sharpening**
![image](https://user-images.githubusercontent.com/9552382/173771862-424c877f-33c7-4317-820c-5e1dfbf319b3.png)
**Longer training**
![image](https://user-images.githubusercontent.com/9552382/173771921-f20ac3a2-7e7d-4a2d-8fac-e1264cedcd48.png)
